theme/color.go

```diff
@@ -1,11 +1,48 @@
 package theme
 
-import "image/color"
+import (
+	"fmt"
+	"image/color"
+	"strings"
+)
 
 // Brighten takes a color and brightens / dims it by the given
-// factor. Each value in the factor represents a 10% increase
-// in the intensity of the color.  Negative values allow for 
-// dimming the color
-func Brighten(c color.Color, factor float64) color.Color {
+// amount. Each value in the amount represents a 10% increase
+// in the intensity of the color.  Negative values allow for
+// dimming the color. This is used to compute a color such as
+// a hightlight on hover based on a base color.
+func Brighten(clr color.Color, amount float64) color.Color {
+	factor := 1.0 + (amount / 10.0)
+	c, ok := clr.(color.NRGBA)
```

Comment:
Why only support brigtening of NRGBA this might be surprising for a public API and it's not documented?
Why not get the channels from `clr.RGBA()`?

Reply:
just discovered the beauty of clr.RGBA() - yep, will refactor all this later on